spark-screenshot-testing/src/test/kotlin/com/adevinta/spark/popover/PopoverScreenshot.kt

```diff
@@ -0,0 +1,123 @@
+import org.junit.runner.RunWith
+
+@OptIn(ExperimentalMaterial3Api::class, ExperimentalSparkApi::class)
+@RunWith(TestParameterInjector::class)
+internal class PopoverScreenshot {
+
+    private val enableList: List<Boolean> = listOf(true, false)
+
+    @get:Rule
+    val paparazzi = Paparazzi(
+        maxPercentDifference = MaxPercentDifference,
+        theme = PaparazziTheme,
+        renderingMode = SessionParams.RenderingMode.SHRINK,
+        showSystemUi = true,
+        environment = patchedEnvironment(),
+        deviceConfig = DeviceConfig.PIXEL_C.copy(
+            softButtons = false,
+            locale = "fr-rFR",
+        ),
+    )
+
+    @Test
+    fun test() {
+        paparazzi.sparkSnapshot(name = "Popover") {
+            val state = rememberTooltipState(isPersistent = true)
+            val scope = rememberCoroutineScope()
+
+            Row {
+                enableList.forEach { isEnabled ->
+                    Column(
+                        verticalArrangement = Arrangement.spacedBy(8.dp),
+                    ) {
+                        Text(
+                            text = "isDismissButtonEnabled: $isEnabled",
+                        )
+                        Row {
+                            Popover(
+                                popoverContent = {
+                                    Column {
+                                        Text(
+                                            text = "Title",
+                                            modifier = Modifier.padding(bottom = 16.dp),
+                                            style = SparkTheme.typography.headline1.copy(fontWeight = FontWeight.Bold),
+                                        )
+                                        Text(
+                                            text = "Do you want to have this cookie now?",
+                                            modifier = Modifier.padding(bottom = 16.dp),
+                                            style = SparkTheme.typography.body2.copy(fontWeight = FontWeight.Bold),
+                                        )
+                                        Text(
+                                            text = "Text Link",
+                                            textDecoration = TextDecoration.Underline,
+                                            style = SparkTheme.typography.body1.copy(fontWeight = FontWeight.Bold)
+                                                .copy(color = SparkTheme.colors.accent),
+                                        )
+                                    }
+                                },
+                                isDismissButtonEnabled = isEnabled,
+                                popoverState = state,
+
+                            ) {
+                                ButtonOutlined(
+                                    text = "Display Popover",
+                                    onClick = { scope.launch { state.show() } },
```

Comment:
For screenshot it should be more useful to directly show the popover as we can't interact with it like in the catalog app so we always get only the button that should show it

Reply:
@soulcramer 
I’ve made a solution for this comment on the last commit
But it didn’t get caught still in Paparazzi screenshot
Do you suggest something for it?

Reply:
hum, I see we have the same issue with other components displayed in a different window, we will have to figure out a way to make them display correctly